frontend/src/styles/App.css

```diff
@@ -107,3 +107,21 @@ footer {
 .card-body {
   background-color: var(--light-color) !important;
 }
+
+.transactionContainer{
+  background-color: var(--light-color) !important;
+  margin-top: 8px !important; 
+}
+
+.transactionContainer *{
+  font-size: 16px !important;
+  
+}
+
+.transactionContainer button{
+height: 32px !important;
+width: 32px !important;
+margin: -8px;
+margin-bottom: 8px;
+padding: 3px;
+} 
```

Comment:
We should avoid adding more custom button styling, the element will just start behaving oddly (e.g. needs those hacky margin and padding styles to align properly and the font-size changing on hover).

I removed some of the input/select/button related stylings to clean this up a bit - please check if it seems ok.